src/main/java/com/exasol/exasoltestsetup/testcontainers/ExasolTestcontainerTestSetup.java

```diff
@@ -22,7 +22,7 @@
  */
 public class ExasolTestcontainerTestSetup implements ExasolTestSetup {
```

Comment:
Maybe we could move ExasolTestcontainerTestSetup and related classes (ExasolTestSetup, ExasolTestSetupFactory) to test packages? It seems that these classes are only used in Tests.
Or are they used in other modules?
So I guess, if these classes are only used in tests and are not used in other modules, then we could move them to test packages.

Reply:
Good point! But these classes are meant for production use, they are the reason for this project ;)